com.tracktion.Waveform11.yml

```diff
@@ -0,0 +1,137 @@
+id: com.tracktion.Waveform11
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform11
+finish-args:
+  - --device=all
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2
+  - --env=LXVST_PATH=/app/extensions/Plugins/lxvst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=home/.vst:create
```

Comment:
you probably also want `~/.lxvst` as it's the other path for VST(2) on Linux

Reply:
Excellent point. Added.